spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/mapping/FirestoreMappingContext.java

```diff
@@ -35,6 +36,12 @@
 		AbstractMappingContext<FirestorePersistentEntity<?>, FirestorePersistentProperty>
 		implements ApplicationContextAware {
 
+	private FirestoreClassMapper firestoreClassMapper;
```

Comment:
It doesn't seem like this class is doing anything with the `firestoreClassMapper` aside from just holding it. So, why should it be here?

Reply:
there is a usage in PartTreeFirestoreQuery.